packages/storybook/stories/ButtonV2.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Button } from "@dailydotdev/shared/src/components/buttons/ButtonV2";
+import PlusIcon from "@dailydotdev/shared/src/components/icons/Plus";
+
+const meta: Meta<typeof Button> = {
+  component: Button,
+  parameters: {
+    controls: {
+      expanded: true,
+    },
+  },
+  argTypes: {
+    href: {
+      control: "text",
+    },
+    onClick: {
+      control: "object",
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Button>;
+
+export const Basic: Story = {
```

Comment:
Yeah that for sure, I was more on having this as an additional thing rather then replacement of what we have now. Or we might want to keep in docs all the components rendered and then in the story have it customisable. It would need some work though this was since at this moment we are using auto generation.

Reply:
Great suggestion for a follow up task for someone when they are completely stuck :)